src/libs/ajax/DataRepo.ts

```diff
@@ -165,7 +165,7 @@ export interface DataRepoContract {
     createSnapshotRequest(request: DatasetAccessRequest): Promise<DatasetAccessRequestApi>;
     getCounts(request: DatasetParticipantCountRequest): Promise<DatasetParticipantCountResponse>;
     getConcepts(parent: SnapshotBuilderConcept): Promise<GetConceptsResponse>;
-    getConceptsHierarchy(concept: SnapshotBuilderConcept): Promise<GetConceptsHierarchyResponse>;
+    getConceptHierarchy(concept: SnapshotBuilderConcept): Promise<GetConceptsHierarchyResponse>;
```

Comment:
Should the type of the response become `GetConceptHierarchyResponse`?

Reply:
Yes, good catch!